vanetza/asn1/tests/asn1c_wrapper.cpp

```diff
@@ -56,6 +56,32 @@ TEST(asn1c_wrapper, validate) {
     EXPECT_FALSE(msg.empty());
 }
 
+TEST(asn1c_wrapper, compare) {
+    test_wrapper wrapper1(asn_DEF_VanetzaTest);
+    OCTET_STRING_fromString(&wrapper1->string, "1234");
+    test_wrapper wrapper2(asn_DEF_VanetzaTest);
+    OCTET_STRING_fromString(&wrapper2->string, "1234");
+    test_wrapper wrapper3(asn_DEF_VanetzaTest);
+    OCTET_STRING_fromString(&wrapper3->string, "0123");
+    test_wrapper wrapper4(asn_DEF_VanetzaTest);
+    wrapper4->field = 5;
+    OCTET_STRING_fromString(&wrapper4->string, "1234");
+
+    // .compare()
+    EXPECT_TRUE(!wrapper1.compare(wrapper1));
```

Comment:
I think the tests should check the actual integer value here.

Reply:
Thanks for the input. Do you mean add additional `EXPECT_*`s for wrappers with only the integer field populated?

Reply:
The newly added `compare` method returns an integer but the tests here treat the return value only as a boolean.
I suggest to replace `EXPECT_TRUE` by `EXPECT_EQ(wrapper1.compare(wrapper1), 0)` etc.